integrations/aws-ecs-monitoring.mdx

```diff
@@ -38,80 +49,296 @@ For Each ECS Task that you want to monitor, add this one sidecar container in yo
+            "name": "MW_API_KEY",
+            "value": "<Your MW API Key here>"
+            },
+            {
+            "name": "MW_TARGET",
+            "value": "<Your MW Target here>"
+            }
+        ],
+        "mountPoints": [
+            {
+            "sourceVolume": "docker-sock",
+            "containerPath": "/var/run/docker.sock",
+            "readOnly": false
+            }
+        ]
+        }
+    ],
+    "volumes": [
+        {
+        "name": "docker-sock",
+        "host": {
+            "sourcePath": "/var/run/docker.sock"
+        }
+        }
+    ],
+    "family": "mw-agent-task"
+    }
+    ```
+    </Accordion>
+
+    <Accordion title="Step 2: Register your Task Definition File">
+        You can register your Task Definition File using the the AWS CLI or their web UI.
+
+        <Accordion title="AWS CLI">
+            Execute the following command to register your Task Definition File in AWS. Learn more about the Amazon ECS CLI [here](https://docs.aws.amazon.com/cli/latest/reference/ecs/).
+            
+            ```cli CLI
+            aws ecs register-task-definition --cli-input-json file://<mw-ecs-agent.json>
+            ```
+        </Accordion>
+
+        <Accordion title="AWS Web UI">
+            Complete the following steps in the AWS Management Console to register your Task Definition File.
+
+            1. Log in to your AWS Management Console and navigate to the Elastic Container Service (ECS) section
+            2. In the left-hand menu, click on Task Definitions
+            3. Create a new Task Definition under the JSON tab
+            4. Copy and paste the configuration from your Task Definition file
+            5. Save your Task Definition JSON file
+            6. Click Create to register the Task Definition in AWS 
+
+        </Accordion>
+
+        <Note> Database integrations for MW Agent are currently not supported on ECS. </Note>
+            </Accordion>
+
+    <Accordion title="Step 3: Schedule the MW Agent as a Daemon Service">
+
+    Set up the Middleware Agent Task Definition as a Daemon Service to ensure only one MW Agent container is running on each EC2 instance of the ECS cluster
+
+    1. Log in to your AWS Management Console and navigate to the Elastic Container Service (ECS) section
+
+    2. Choose the ECS cluster you will run the MW Agent on
+
+    3. Create a new service within the selected cluster
+
+    4. Setup the Environment section
+
+        a. Select Launch type under compute options
+
+        b. Select EC2 as the Launch type
+
+    5. Setup the Deployment Configureation section
+
+        a. Select Service as the Application type 
+
+        b. Specify the Task Definition you created earlier under the Family section 
+
+        c. Provide a unique Service Name 
+        
+        d. Choose DAEMON as the Service Type
+
+    6. Proceed to create the service
+    </Accordion>
+
+
+
+</AccordionGroup>
+
+
+# Container Trace and Log Collection
```

Comment:
Is this EC2 only, or Fargate as well? If EC2 only let's make it a subheading to the EC2 stuff above.